base/float.jl

```diff
@@ -541,7 +541,7 @@ function float{T}(A::AbstractArray{T})
     convert(AbstractArray{typeof(float(zero(T)))}, A)
 end
 
-for fn in (:float,:big)
+for fn in (:float,)
```

Comment:
You could probably just deloopify this since it's iterating over a single value, or so I would assume.